docs/pages/web/tabs.js

```diff
@@ -343,15 +376,15 @@ function TabExample() {
           defaultCode={`
 function TabExample() {
   const [activeIndex, setActiveIndex] = React.useState(0);
-  const [isTransparent, setIsTransparent] = React.useState(true);
+  const [isTransparent, setIsTransparent] = React.useState(false);
```

Comment:
Somewhat unrelated — I think it's more effective to show the problem first, then allow the user to "solve" the problem by interacting with the switch. This is how we've handled the Wrapping example and it seems better to me.